Client/Application/LeanplumIntegration.swift

```diff
@@ -39,6 +39,7 @@ enum UserAttributeKeyName: String {
     case focusInstalled = "Focus Installed"
     case klarInstalled = "Klar Installed"
     case signedInSync = "Signed In Sync"
+    case mailtoIsDefault = "Mailto Is Default"
```

Comment:
@bkmunar Wait, don't you need to start using this "mailtoIsDefault" UserAttributeKeyName below? It looks like you're still using the same "alternateMailClient" key name.